### PR TITLE
Update Brand logos page

### PR DESCRIPTION
**Pull request by abovedave**
An updated page for company logos, currently [located here](https://stackoverflow.com/company/logos) (and some one ones).

I think it makes sense to consolidate these into Stacks, rather than living in two different places. As far as I can tell the existing logo page is not linked from anywhere - I will setup a redirect anyway when the time comes.

### CODE REVIEW

File: dist/js/stacks.js [hunk 1]
```diff
@@ -1757,7 +1757,7 @@ Copyright © 2019 Basecamp, LLC
 ;
 
 /**
- * @popperjs/core v2.4.0 - MIT License
+ * @popperjs/core v2.4.4 - MIT License
```
Comment (aaronshekey) [inline dist/js/stacks.js]: 🤔 This probably shouldn't be bumped in _this_ PR.

File: docs/brand/logo.html [hunk 2]
```diff
@@ -43,68 +43,183 @@
...
+        </div>
+    </div>
+
+    {% header "h3", "Stack Exchange" %}
+    <div class="grid gs32 gsx fw-wrap mt24">
+        <div class="grid--cell mb12">
+            <img class="h48" src="{{ "/assets/img/logos/sf/sf-logo.svg" | relative_url }}" />
+            <div class="grid gsx gs4 ai-center fc-light fs-fine mt16 mb32">
+                <div class="grid--cell">Download:</div>
+                <a href="{{ "/assets/logos/sf/sf-logo.svg" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">SVG</a>
```
Comment (aaronshekey) [inline docs/brand/logo.html]: These links to Server Fault are 404ing for me.

File: docs/brand/logo.html [hunk 3]
```diff
@@ -43,68 +43,183 @@
...
+                <a href="{{ "/assets/logos/sf/sf-logo.png" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">PNG</a>
+                <a href="{{ "/assets/logos/sf/sf-logo.eps" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">EPS</a>
+            </div>
+        </div>
+        <div class="grid--cell mb12">
+            <img class="h48" src="{{ "/assets/img/logos/sf/sf-icon.svg" | relative_url }}" />
+            <div class="grid gsx gs4 ai-center fc-light fs-fine mt16 mb32">
+                <div class="grid--cell">Download:</div>
+                <a href="{{ "/assets/logos/sf/sf-icon.svg" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">SVG</a>
```
Comment (aaronshekey) [inline docs/brand/logo.html]: 404 on these

File: docs/brand/logo.html [hunk 4]
```diff
@@ -43,68 +43,183 @@
...
+                <a href="{{ "/assets/logos/sf/sf-icon.png" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">PNG</a>
+            </div>
+        </div>
+        <div class="grid--cell mb12">
+            <img class="h48" src="{{ "/assets/img/logos/su/su-logo.svg" | relative_url }}" />
+            <div class="grid gsx gs4 ai-center fc-light fs-fine mt16 mb32">
+                <div class="grid--cell">Download:</div>
+                <a href="{{ "/assets/logos/su/su-logo.svg" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">SVG</a>
```
Comment (aaronshekey) [inline docs/brand/logo.html]: 404 here

File: docs/brand/logo.html [hunk 5]
```diff
@@ -43,68 +43,183 @@
...
+                <a href="{{ "/assets/logos/su/su-logo.png" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">PNG</a>
+                <a href="{{ "/assets/logos/su/su-logo.eps" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">EPS</a>
+            </div>
+        </div>
+        <div class="grid--cell mb12">
+            <img class="h48" src="{{ "/assets/img/logos/su/su-icon.svg" | relative_url }}" />
+            <div class="grid gsx gs4 ai-center fc-light fs-fine mt16 mb32">
+                <div class="grid--cell">Download:</div>
+                <a href="{{ "/assets/logos/su/su-icon.svg" | relative_url }}" class="grid--cell s-btn s-btn__filled s-btn__xs">SVG</a>
```
Comment (aaronshekey) [inline docs/brand/logo.html]: 404 here

**Review by aaronshekey — CHANGES_REQUESTED** [review]
THANK YOU

1. Revert the popper update, I'll do that separately.
2. Couple 404s down in the Serverfault section
3. `teams-logo-compact.png` has a goofy "O" in "FOR".

File: dist/js/stacks.js (hunk 1 above)
Comment (abovedave) [reply dist/js/stacks.js]: Interesting that happened automatically

Comment (aaronshekey) [reply dist/js/stacks.js]: Hey, no judgement. I bet `npm install` trolled you here.

**Review by aaronshekey — APPROVED** [review]

**Pull request merged**